src/mscorlib/ref/mscorlib.cs

```diff
@@ -7344,7 +7344,7 @@ public enum AssemblyNameFlags
     [System.Runtime.InteropServices.ComVisibleAttribute(true)]
     public class AssemblyNameProxy : System.MarshalByRefObject
     {
-        internal AssemblyNameProxy() { }
+        public AssemblyNameProxy() { }
```

Comment:
@weshaggard Should I be removing `AssemlbyNameProxy` from here and have it implemented on corefx side?